### PR TITLE
Fix (dictionary check all sections on adding word)

### PR DESCRIPTION
**Pull request by Aca-jov**
SD-2786

### CODE REVIEW

**Comment by amagdas** [comment]
An e2e test I think is needed for this, the basic setup should be done already for undo/redo functionality.

File: client/app/scripts/superdesk-dictionaries/dictionaries.js [hunk 1]
```diff
@@ -142,7 +142,7 @@
         function addWordToUserDictionary(word, lang) {
             return getUserDictionary(lang).then(function(userDict) {
                 var words = userDict.content || {};
-                words[word] = words[word] ? words[word] + 1 : 1;
+                words[word.toLowerCase()] = words[word.toLowerCase()] ? words[word.toLowerCase()] + 1 : 1;
```
Comment (plamut) [inline client/app/scripts/superdesk-dictionaries/dictionaries.js]: Calling `toLowerCase()` multiple times is redundant, better to call it only once in store the result in a temporary variable.

File: client/app/scripts/superdesk/editor/editor.js [hunk 2]
```diff
@@ -487,10 +489,14 @@ angular.module('superdesk.editor', [])
                 };
 
                 scope.addWordToDictionary = function() {
-                    var editorElem = elem.find(scope.type === 'preformatted' ?  '.editor-type-text' : '.editor-type-html');
                     var word = scope.replaceTarget.textContent;
                     spellcheck.addWordToUserDictionary(word);
-                    editorElem.trigger('input');
+
+                    for (var i = 0; i < editor.elements.length; i++){
+                        var editorElem = editor.elements[i].find(
```
Comment (plamut) [inline client/app/scripts/superdesk/editor/editor.js]: Perhaps consider using `forEach()` here for looping through the Array, more concise:
```js
editor.elements.forEach(function (editorEelem) {
    ...
});
```

File: client/app/scripts/superdesk/editor/spellcheck/spellcheck.js [hunk 3]
```diff
@@ -257,7 +257,7 @@ function SpellcheckService($q, api, dictionaries, editor) {
      */
     this.addWordToUserDictionary = function(word) {
         dictionaries.addWordToUserDictionary(word, lang);
-        dict.content[word] = dict.content[word] ? dict.content[word] + 1 : 1;
+        dict.content[word.toLowerCase()] = dict.content[word.toLowerCase()] ? dict.content[word.toLowerCase()] + 1 : 1;
```
Comment (plamut) [inline client/app/scripts/superdesk/editor/spellcheck/spellcheck.js]: Similar to the other comment above (multiple calls to `toLowerCase()`).

**Comment by pavlovicnemanja92** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**